src/extension/background-script/connectors/alby.ts

```diff
@@ -254,11 +254,18 @@ export default class Alby implements Connector {
         token: this.config.oAuthToken, // initialize with existing token
       });
 
+      authClient.on("tokenRefreshed", async (token: Token) => {
+        await this._updateOAuthToken(token);
+      });
+
+      authClient.on("tokenRefreshFailed", async (error: Error) => {
+        console.error("Failed to Refresh token", error);
+      });
+
       if (this.config.oAuthToken) {
         try {
           if (authClient.isAccessTokenExpired()) {
-            const token = await authClient.refreshAccessToken();
-            await this._updateOAuthToken(token.token);
+            await authClient.refreshAccessToken();
```

Comment:
Further down in this file (Lines 290-292)
```
 const token = await authClient.requestAccessToken(code);
      await this._updateOAuthToken(token.token);
      return authClient;
```
We should not need to manually call this._updateOAuthToken. It should be fired from the `tokenRefreshed` callback (needs change in alby-js-sdk).

Reply:
made a pr for that https://github.com/getAlby/js-sdk/pull/121

Reply:
We decided this is not a good way to go, so let's leave this as-is